package.json

```diff
@@ -76,16 +62,15 @@
     "server:prod:ci": "http-server dist -p 8080 -c-1 --cors",
     "server:staging": "cross-env PORT=8080 NODE_ENV=production npm run webpack:server -- --config config/webpack.server.js  --progress --profile --bail",
     "server:wds": "npm run webpack-dev-server -- --config config/webpack.server.js --open --progress --profile --watch --content-base src/",
-    "server:createindexes": "mongo --host localhost ./src/server/test/manual/createIndexes.js",
+    "server:createindexes": "mongo --host localhost ./src/server/scripts/createIndexes.js",
     "start": "node --max_old_space_size=2000  ./dist/server.js",
-    "start:mongo": "sh start-mongo.sh '/var/lib/mongodb/'",
-    "start:mongo:windows": "sh start-mongo.sh 'C:\\data\\db'",
+    "start:mongo": "sh scripts/start-mongo.sh '/var/lib/mongodb/'",
+    "start:mongo:windows": "sh scripts/start-mongo.sh 'C:\\data\\db'",
     "start:prod": "npm run watch:server:prod",
     "start:hmr": "npm run server:dev:hmr",
-    "test": "npm run lint && karma start",
+    "test": "npm run lint",
```

Comment:
Does this mean that after this change there are no automated tests being run?

Reply:
Hi @jay-hodgson, I need to add the tests back (this should be done by Friday). I temporary removed karma test so we can test the Travis pipeline / EB deployment